include/sc2api/sc2_typeenums.h

```diff
@@ -866,6 +867,7 @@ namespace sc2 {
         CARRYHARVESTABLEVESPENEGEYSERGAS = 273,
         CARRYHARVESTABLEVESPENEGEYSERGASPROTOSS = 274,
         CARRYHARVESTABLEVESPENEGEYSERGASZERG = 275,
+        RAVENSCRAMBLERMISSILE = 277,
```

Comment:
Please, add the new ID to the `BuffIDToName` function (`src/sc2api/sc2_typeenums.cc`).